relval scripts update and adding new 2015C 25ns data workflows

hengne opened the pull request:
1.) adding new 2015C 25ns data workflows, including json file update, refer to branch: hengne:hengne:from-CMSSW_7_5_2_relval_newData_fullList
2.) update mixing/premixing inputs in RelVal scripts
3.) change to use the three new BPH workflows as requested by the BPH group.

hengne committed:
use 1340.1,1348.1,1351.1 to officially replace the previous 1340.0,1348.0,1351.0 as requested by the BPH group

hengne committed:
update fastSim minbias for mixing

hengne committed:
update fullsim premixing input

hengne committed:
update fastSim premixing input

hengne committed:
json file for 2015 run2 data update according to HN https://hypernews.cern.ch/HyperNews/CMS/get/physics-validation/2474.html

hengne committed:
add 2015c new data relval workflows, ported from 75x according to branch hengne:hengne:from-CMSSW_7_5_2_relval_newData_fullList

hengne committed:
update 2015b workflow numbering similarly to 2015c

hengne committed:
fix LumiList selection

hengne committed:
fix conflict after rebase

cmsbuild commented:
A new Pull Request was created by @hengne (Hengne Li) for CMSSW_7_6_X.

relval scripts update and adding new 2015C 25ns data workflows

It involves the following packages:

Configuration/PyReleaseValidation
DPGAnalysis/Skims


@cmsbuild, @srimanob, @boudoul, @franzoni can you please review it and eventually sign? Thanks.
@ghellwig, @Martin-Grunewald this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@degano you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

franzoni commented:
hello @hengne ,
thanks a lot for this.

I have a couple of suggestions:

+ can you please add, for 2015C, the processing of these two raw-reco skims :
  /DoubleEG/Run2015C-ZElectron-PromptReco-v1/RAW-RECO
  /SingleMuon/Run2015C-ZMu-PromptReco-v1/RAW-RECO
  ?
  Given their reduced rate w.r.t. to the PD they come from, the runs in input can be extended to all 3 2015C certified runs we have so far 

+ ragarding : https://hypernews.cern.ch/HyperNews/CMS/get/dataopsrequests/8090/1.html 
   could you please change /DoubleElectron/Run2012D-ZElectron-PromptSkim-v1/RAW-RECO 
   with /DoubleElectron/Run2012D-ZElectron-22Jan2013-v1/RAW-RECO ?

Thanks !

cmsbuild commented:
+1
Tested at: f723f597ccd3b975bc6f5e06ebbe8d1160861098
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7921/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7921/summary.html

hengne committed:
change /DoubleElectron/Run2012D-ZElectron-PromptSkim-v1/RAW-RECO
with /DoubleElectron/Run2012D-ZElectron-22Jan2013-v1/RAW-RECO

hengne committed:
add 2 new 2015c 25ns workflows

hengne committed:
merge branch with 2 new relvals for 2015c.
Merge branch 'from-CMSSW_7_6_0_pre4_relval_newdata' into from-CMSSW_7_6_0_pre4_relval_newdata_rebase

hengne committed:
rebase-
Merge branch 'CMSSW_7_6_X' into from-CMSSW_7_6_0_pre4_relval_newdata_rebase

hengne commented:
@franzoni Hi Giovanni, I have implemented what you suggested. Please test again.
Thanks, hengne

cmsbuild commented:
Pull request #11127 was updated. @cmsbuild, @srimanob, @boudoul, @franzoni can you please check and sign again.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

davidlt commented on Configuration/PyReleaseValidation/python/relval_steps.py:
```diff
@@ -138,6 +138,20 @@
+# Run2015C, 25ns: 254790 (852 LS and 65 files), 254852 (126 LS and 5 files), 254879 (178 LS and 11 files)
+Run2015C=selectedLS([254790])
+Run2015C_full=selectedLS([254790, 254852, 254879])
+steps['RunHLTPhy2015C']={'INPUT':InputInfo(dataSet='/HLTPhysics/Run2015C-v1/RAW',label='hltPhy2015C',events=100000,location='STD', ls=Run2015C)}
```
Is this correct? IIUC, this would run 100K events in IB RelVals. Hopefully I am wrong. Otherwise this should be set to CAF location.

davidlt replied:
Ah, all fine. It sets 100 events for STD somewhere in matrix.

hengne replied:
Hi  @davidlt 
I just checked the InputInfo function in MatrixUtil.py. Unless I misunderstood the codes, seems to me that this parameter "events" is actually not used in its return string.

cmsbuild commented:
+1
Tested at: 9f11b8bf439af07201dbb68666b7a6ad54727417
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7958/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/9f11b8bf439af07201dbb68666b7a6ad54727417
https://github.com/cms-sw/cmssw/commit/ae26cd678dab3d20a7f2a7f1d11af6860f02f121
https://github.com/cms-sw/cmssw/commit/f545ad8c549260a724d5c675a5b8160b9623bdb8
https://github.com/cms-sw/cmssw/commit/37dd994a27d04cfa77fa278a63028887372e2b3c
https://github.com/cms-sw/cmssw/commit/0253050b5a90aa18841ab714295ed47df1e2a110
https://github.com/cms-sw/cmssw/commit/d5d4133ab96d16723b249c1bcfca5475cad35272
https://github.com/cms-sw/cmssw/commit/a6ee6096a1ddad61f1742a1d46bfd40fc9c79bdc
https://github.com/cms-sw/cmssw/commit/bb4924e00a12d64f318e212fb94f31fab8de0989
https://github.com/cms-sw/cmssw/commit/310a4496f1c4800cd81d2c49c96f3b3c340b06ac
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7958/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7958/git-merge-result

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7958/summary.html

hengne committed:
temporary commit out the QQH1352TINPUT line, will put back when gen-sim samples are generated

hengne committed:
replace deprecated input dataset for two 2012 data workflow

hengne committed:
Merge branch 'from-CMSSW_7_6_0_pre4_relval_newdata' into from-CMSSW_7_6_0_pre4_relval_newdata_rebase

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

hengne committed:
rebase again
Merge branch 'CMSSW_7_6_X' into from-CMSSW_7_6_0_pre4_relval_newdata_rebase

cmsbuild commented:
Pull request #11127 was updated. @cmsbuild, @srimanob, @boudoul, @franzoni can you please check and sign again.

cmsbuild commented:
+1
Tested at: 9f11b8bf439af07201dbb68666b7a6ad54727417
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7983/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7983/summary.html

franzoni commented:
hello @hengne , does this contain all what you wanted to release ?

hengne commented:
Hi Giovanni, @franzoni , thanks for asking, yes, i have all what we agreed
to include included in this PR now.
Cheers, Hengne

On Wed, Sep 9, 2015 at 7:35 PM, Giovanni Franzoni <notifications@github.com>
wrote:

> hello @hengne <https://github.com/hengne> , does this contain all what
> you wanted to release ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11127#issuecomment-138985851>.
>

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 52b584ab1ec3cda0ee124b2ca0baa3f581bc4275
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7989/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/52b584ab1ec3cda0ee124b2ca0baa3f581bc4275
https://github.com/cms-sw/cmssw/commit/0af44915a8dadbec596c0875be0ee57b8ae8d650
https://github.com/cms-sw/cmssw/commit/800dc34bebf7bd4857b5e0827d0f5d17e6f7a2f5
https://github.com/cms-sw/cmssw/commit/070bdb6294a784445199f89f2c0c8ebf236eb11c
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7989/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7989/git-merge-result

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11127/7989/summary.html

franzoni commented:
+1

all workflows of the standard set:

runTheMatrix.py --what standard --command='-n 1' -i all -j 2

run succesfully w/ this PR.

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_7_6_X IBs  (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @degano, @smuzaffar

davidlange6 commented:
+1
